parquet-avro/src/test/java/org/apache/parquet/avro/TestReadWrite.java

```diff
@@ -117,19 +120,52 @@ public void testEmptyArray() throws Exception {
     }
   }
 
+  @Test
+  public void testEmptyArrayLocal() throws Exception {
```

Comment:
This duplicates `testEmptyArray()`. Can we parameterize them to share common code?